src/main/java/frc/robot/RobotContainer.java

```diff
@@ -92,16 +96,31 @@ private void controllerBindings() {
       // Check which controller is being used
       boolean isXbox = (Math.abs(controlXbox.getY(Hand.kLeft))
           + Math.abs(controlXbox.getX(Hand.kRight))) > (Math.abs(joystickLeft.getY()) + Math.abs(joystickRight.getY()));
-
       // Set the drive powers based on which controller is being used
       if (isXbox) {
         tankSubsystem.setCarDrivePowers(-controlXbox.getY(Hand.kLeft), controlXbox.getX(Hand.kRight));
-      } else {
-        tankSubsystem.setTankDrivePowers(-joystickLeft.getY(), -joystickRight.getY());
+    }
+    else {
+      tankSubsystem.setTankDrivePowers(-joystickLeft.getY(), -joystickRight.getY());
+    }
+
+   if(controlXbox2.getAButtonReleased()){
+      if(clawSubsystem.clawIsOpen){
+        clawSubsystem.closeClaw();
       }
-    };
-    tankSubsystem.setDefaultCommand(new RunCommand(tank, tankSubsystem));
-  }
+      else{
+        clawSubsystem.openClaw();
+      }   
+    }
+    if(controlXbox2.getBButtonReleased()){
+      if(clawSubsystem.clawIsLifted){
+        clawSubsystem.lowerClaw();
+      }
+      clawSubsystem.liftClaw();
+    }
+  };
+  tankSubsystem.setDefaultCommand(new RunCommand(tank, tankSubsystem));
+}

```

Comment:
idk what happened but the indenting here got real funky.

vscode has auto formatting too: https://code.visualstudio.com/docs/cpp/cpp-ide#_code-formatting